package.json

```diff
@@ -0,0 +1,97 @@
+{
+  "name": "api-linting-service",
+  "version": "1.0.0",
+  "description": "",
+  "author": "",
+  "private": true,
+  "license": "APACHE 2.0",
```

Comment:
NPM uses SPDX license codes in this field, for Apache 2.0 this should be `Apache-2.0`

Reply:
done